lib/Indicator.vala

```diff
@@ -40,6 +40,11 @@ public abstract class Wingpanel.Indicator : GLib.Object {
      */
     public string code_name { get; construct; }
 
+    /**
+     * Set the indicator positioning on the panel
```

Comment:
```suggestion
     * Sets the indicator positioning on the panel.
```

Tiny thing but other existing properties use the Third Person form and have the period so let's honor them.